dialog/lib/dialog.ts

```diff
@@ -203,7 +203,8 @@ export class Dialog extends LitElement {
    *
    * Defaults to grow-down.
    */
-  @property({reflect: true}) transition = 'grow-down';
+  @property({reflect: true})
+  transition: 'grow'|'shrink'|'grow-down'|'grow-up'|'grow-left'|'grow-right' = 'grow-down';
```

Comment:
Can you make this a type at the top of the file?

```ts
export type DialogTransition = 'grow'|'shrink'|'grow-down'|'grow-up'|'grow-left'|'grow-right';
```

Then re-export it in `dialog/dialog.ts` for easier use by clients.

```ts
export {DialogTransition} from './lib/dialog';
```

Reply:
Ok, done. Wdyt?